src/app/vm-logs/containers/vm-logs-filter.container.ts

```diff
@@ -136,6 +136,7 @@ export class VmLogsFilterContainerComponent extends WithUnsubscribe()
 
   public ngOnInit() {
     this.store.dispatch(new vmActions.LoadVMsRequest());
+    this.store.dispatch(new UserTagsActions.LoadUserTags());
```

Comment:
Why do you decide to download user tags here? They loaded through home component. So it should work on every page of the app.